index.js

```diff
@@ -26,7 +26,6 @@ module.exports.Comment = require('./lib/comment');
 
 // Compatibility synonyms
 Document.fromXmlString = Document.fromXml;
-Document.fromHtmlString = Document.fromHtmlString;
```

Comment:
My vote would be for `fromHtml`, to keep the parallel:
~~~~
Document.fromHtmlString = Document.fromHtml;
~~~~